Curve-finance-suite/Unkill/src/agent.spec.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
```

Comment:
`FindingType`, `FindingSeverity`, and `Finding` don't need to be imported since they are not being used.